src/docs/product/sentry-basics/sampling/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Sampling
+sidebar_order: 25
+redirect_from:
+  - /product/sentry-basics/metrics/
+description: "Learn more about the different ways that you can sample the transaction events that you send to Sentry."
+---
+
+<Note>
+
+This feature is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).
```

Comment:
```suggestion
This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
```